src/test/scala/org/broadinstitute/dsde/rawls/integrationtest/IntegrationTestBase.scala

```diff
@@ -90,7 +90,7 @@ trait IntegrationTestBase extends FlatSpec with ScalatestRouteTest with Matchers
 
   def workspaceServiceWithDbName(dbName: String) = {
     // setup DB. if it already exists, drop and then re-create it.
-    val dbUrl = s"remote:${orientServer}/${dbName}"
+    val dbUrl = orientUrl.replace("rawls", dbName)   // TODO: don't hard-code this
```

Comment:
So the integration tests write to their own DB (which they wipe) instead of Dev.